tests/test_evaluate_mesh_on_grants.py

```diff
@@ -5,14 +5,42 @@
 from sklearn.preprocessing import MultiLabelBinarizer
 import pandas as pd
 
-from grants_tagger.models.mesh_cnn import MeshCNN
+from grants_tagger.models.mesh_xlinear import MeshXLinear
 from grants_tagger.evaluate_mesh_on_grants import evaluate_mesh_on_grants
 
 
 TRAIN_DATA = [
```

Comment:
Is this the same data as in test_cli.py? (See my comment above)

Reply:
yes it is.. actually it might make sense after all to load it in :D